### ISSUE TITLE
Enable all output cards

### ISSUE THREAD

**Issue by Torinde** [crazii/SBEMU#59]
It seems currently some of the output cards code is not compiled-in. I think it's good to enable those, at least for user testing:
 * `sc_sbxfi`: Creative X-Fi EMU20KX - #10
 * `sc_midas`: MIDAS library handling - #4
 * `sc_sb16`: Sound Blaster 16 - especially when SBEMU gets more features such as #42 and #43 it will be useful to run it on top of
   - bare metal cards
   - VirtualBox, Bochs, etc.
 * `sc_sbpro`: Sound Blaster 1.0, 1.5, 2.0, Pro
   - bare metal cards
   - NTVDM - like VDMSound
 * `sc_wss`: Windows Sound System compatible cards
 * `sc_ess`: ESS card handling
 * `sc_gus`: Gravis UltraSound
 * `sc_wav`: WAV-file and bitstream output - for testing/debugging in DOS?

If feasible to provide emulated card to Windows, will be useful when SBEMU gets more features such as MIDI, PC Speaker, AWE to have:
 * `sc_winds`: Win32 DirectSound output
 * `sc_winwo`: Windows WavOut sound output

**Comment by crazii** [comment]
SBEMU won't work on any windows system by its nature - windows has its own DPMI host and it have no port trapping, and windows won't launch if there's a DPMI installed under DOS (e.g. HDPMI32i), and there's no need to 'emulate a sound blaster card on a sound blaster card', I think we'll pass all ISA cards since ISA machine usually have SB cards and SBEMU may not run on a ISA machine due to it's CPU power.

There're 2 card that need to be added for sure, sc_sbxfi and sc_ess, since ESS have some crappy OPL support AFTER ES1938/1946 (s) (ESS Solo-1), namely the crappy ESS allegro/maestro cards.

**Comment by Torinde** [comment]
> I think we'll pass all ISA cards since ISA machine usually have SB cards

In general yes, although:
- Physical machines used [REMUS](https://www.vogons.org/viewtopic.php?t=15249), because you may have SB16 and want to emulate GUS or Covox, etc. or vice versa. CPU power of P3 or such will probably be enough?
- VirtualBox SB16 doesn't have MIDI and is generally worse than DOSBox. 
- NTVDM has only SB 2.0, but I don't know if HDPMI32i works there? 
  - https://github.com/Baron-von-Riedesel/HX/issues/50
  - https://github.com/leecher1337/ntvdmx64/issues/296

**Comment by crazii** [comment]
Yes it could be done in the long term, but GUS/Covox emu over SB is not so much needed (couple of decades have passed, few people own ISA machines now, and even fewer of them want a GUS emu, and I doubt few people wanted a GUS emu even decades ago), let's focus on the legacy emu over PCI sound card for now. 

The best step is to make a stable GUS/Covox emu over PCI cards first, and then it would be easier to add ISA cards support. Otherwise new codes of GUS emu + untested ISA drivers would make it extremely hard to debugging, (or maybe to debug the ISA drivers first, using the 'SB emu over SB' to test the ISA drivers). Besides, based on the modification of Mpxplay PCI drivers, I don't think it be that easy to enable ISA drives and make them work.

**Comment by Torinde** [comment]
Enabling Sound blaster drivers may also help to isolate an issue from https://github.com/Baron-von-Riedesel/VSBHDA/issues/8#issuecomment-2039240514:
> No. It reminds me of the "FPU problem" and also that vsbhda still keeps silent in VBox with ICH9.

I haven't tested yet SBEMU with [HDA in Virtualbox](https://github.com/crazii/SBEMU/issues/140#issuecomment-2440089830), but it would be good if it can be tested _also_ with SB - to check if the problem is HDA specific or not.

**Comment by Torinde** [comment]
Related:
- #142 
- #152
